webui/src/lib/components/nav.jsx

```diff
@@ -53,12 +56,60 @@ export const Link = (props) => {
     return React.createElement(RouterLink, linkProps, props.children);
 }
 
-export const NavItem = ({ href, active, children }) => {
+export const NavItem = ({href, active, children}) => {
     return (
         <Nav.Item>
             <Link href={href} component={Nav.Link} active={active}>
                 <>{children}</>
             </Link>
         </Nav.Item>
     );
-};
\ No newline at end of file
+};
+
+export function CodeTabPanel({children, value, index, ...other}) {
```

Comment:
Why is this in nav.jsx?

Reply:
I considered `nav.jsx` as a file that contains all sorts of general-purpose navigation components. 
Although this is not a navigation component, it basically provides a framework to handle "code tabs" (snippets if you will) content in a unified way.
In case you do not find it appropriate, I can take it out

Reply:
IIUC, this is not a standalone component - it needs to be used in a very particular way together with <Tab>, so I wouldn't put this in this package which is general purpose.